package.json

```diff
@@ -60,7 +60,7 @@
     "commitizen": "3.0.5",
     "common-tags": "1.8.0",
     "cross-env": "5.2.0",
-    "cz-lerna-changelog": "2.0.2",
```

Comment:
`cz-lerna-changelog` v2.0.2 does not support node v6, it supports node greater than or equal to v8.30.